src/AllAssignmnets/Assignment2/Assignment2.java

```diff
@@ -0,0 +1,28 @@
+package AllAssignmnets.Assignment2;
+
+
+import java.util.*;
+
+public class Assignment2 {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        String input = scanner.next();
+        System.out.println(checkString(input.toLowerCase()));
+    }
+
+    public static String checkString(String input){
```

Comment:
This is voilating the single responsibility principle. Please follow SOLID principles.

Reply:
solved